api/proto/ci/concourse.proto

```diff
@@ -30,7 +30,7 @@ message ConcourseAccount {
   // (Required) The URL at which your Concourse search is reachable.
   string url = 4;
 
-  // Fiat permissions configuration. A user must have at least one of the READ
+  // Configuration for Fiat permissions. A user must have at least one of the READ
```

Comment:
Trying to derive a rule from the changes here...when should we say "configuration for" vs. "configuration to use"?

Reply:
No rule here; this was just my "ear." :ear: